pkgs/development/python-modules/panda3d/default.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, python
+, isPy37
+, isPy38
+, isPy39
+, autoPatchelfHook
+, libXext
+, libGL
+, gdk-pixbuf
+, cairo
+, pango }:
+
+let pyVerNoDot = builtins.replaceStrings [ "." ] [ "" ] python.pythonVersion;
+    version = "1.10.10";
+    srcs = import ./binary-hashes.nix version;
+    unsupported = throw "Unsupported system";
+in buildPythonPackage rec {
+  pname = "panda3d";
+
+  inherit version;
+
+  format = "wheel";
+
+  src = fetchurl srcs."${stdenv.system}-${pyVerNoDot}" or unsupported;
```

```suggestion
  src = fetchurl (import ./binary-hashes.nix version)."${stdenv.system}-${builtins.replaceStrings [ "." ] [ "" ] python.pythonVersion}" or (throw "Unsupported system";);
```

maybe make it multiline.